tests/README.md

```diff
@@ -113,14 +113,15 @@ Full list of arguments
     dSpeed=   : Minimum decompression speed
     cMem=     : Maximum compression memory
     lvl=      : Searches for solutions which are strictly better than that compression lvl in ratio and cSpeed, 
-    stc=      : When invoked with lvl=, represents percentage slack in ratio/cSpeed allowed for a solution to be considered (Default 99%)
+    stc=      : When invoked with lvl=, represents percentage slack in ratio/cSpeed allowed for a solution to be considered (Default 100%)
               : In normal operation, represents percentage slack in choosing viable starting strategy selection in choosing the default parameters
                 (Lower value will begin with stronger strategies) (Default 90%)
-    preferSpeed= / preferRatio=
-              : Only affects lvl = invocations. Defines value placed on compression speed or ratio
-                when determining overall winner (default speed = 1, ratio = 5 for both, higher = more valued).
+    ratioSpeedTradeoff=#:#
```

Comment:
I would simplify this setting to a single value `speedRatio=#`,
which would effectively be the same as `preferRatio=#` and pin `preferSpeed` to `1`.

Should you worry about a potential lack of available values at lower ranges (1,2,3, etc.),
you may allow fractional values.

Also : pay attention to value `0` : what does it mean ? how is it handled ?

Reply:
Zero just means setting that multiplier to zero, so speed / ratio isn't factored at all when comparing two solutions. I was thinking of making the ratio a single number, but I thought it'd be annoying to add a function to parse doubles just for this parameter.